buildSrc/src/main/groovy/com/uber/okbuck/experimental/bazel/AndroidLibraryRule.groovy

```diff
@@ -0,0 +1,66 @@
+package com.uber.okbuck.experimental.bazel
+
+import com.uber.okbuck.core.model.AndroidTarget
+import com.uber.okbuck.rule.BuckRule
+
+final class AndroidLibraryRule extends BuckRule {
+    private static final String RULE_TYPE = "android_library"
+
+    private final Set<String> srcSet
+    private final String manifest
+    private final Set<String> providedDeps
+    private final String packageName
+    private final Set<AndroidTarget.ResBundle> resources
+
+    AndroidLibraryRule(
```

Comment:
Ok, for documentations sake, the way that annotation processors work in Bazel is we create a [`java_plugin`](https://bazel.build/versions/master/docs/be/java.html#java_plugin) rule that depends on a `java_library` that builds that annotation processor or a `java_import` that wraps a prebuilt JAR. The `java_plugin` specifies a `processor_class`. Then we put the `java_plugin` in the deps of an `android_library` or `java_library` that it should run on.